open-in-remote-terminal@mamolinux/info.json

```diff
@@ -0,0 +1,3 @@
+{
+    "author": "mamolinux"
```

Comment:
This should be the GitHub username of the author/maintainer. @hsbasu Will you be getting notifications for this account?

Reply:
Yeah.